packages/firestore/test/unit/specs/spec_test_runner.ts

```diff
@@ -1566,6 +1567,8 @@ export interface SpecWatchFilter
   '1': string | undefined;
 }
 
+export type SpecLimitType = 'LimitToFirst' | 'LimitToLast';
```

Comment:
Could this use the Enum type that you defined in your previous PR (which already has a String mapping)?

Reply:
Hmm, I am merely following other cases (filter, orderBy) here that they all have separate types instead of using prod types. I thought this was by design?